views/movies/movies.hbs

```diff
@@ -0,0 +1,8 @@
+<h2>Movies List</h2>
+
+{{#each movies}}
+<section>
+    <p>{{this.title}}</p>
+    <a href="/movies/{{_id}}">More Details</a>
+</section>
+{{/each}}
```

Comment:
This code is not working, because you have different variable name in your route. In route you have the name: `celebs`
but in view you are using `movies`

Below is your code from movie routes:

```
// List all movies
router.get("/movies", (req, res, next) => {
    Movie.find()        // empty () - get everything, entire arr
    .then((movieArr) => {
        res.render("movies/movies", {celebs: movieArr})
    })
```